docs/developer-guide/run.md

```diff
@@ -0,0 +1,88 @@
+---
+order: 4
+description: Running a blockchain with Starport.
+---
+
+# Start a Blockchain
+
+Blockchains are decentralized applications.
+
+- In production, blockchains often run the same software on many validator nodes that are run by different people and entities. To launch a blockchain in production, the validator entities coordinate the launch process to start their nodes simultaneously.
```

Comment:
oooh now we're getting into concepts 

put development before production

maybe expand on the whole blockchain bit

we'll need to talk about validators before we jump into these descriptions? 

meaty topic
do we have visuals that show this evolution of a blockchain?